examples/double_pendulum/BUILD.bazel

```diff
@@ -15,13 +15,13 @@ rt_ipynb_test(
     name = "dynamics",
     srcs = ["dynamics.ipynb"],
     size = "medium",
-    data = ["double_pendulum.urdf"],
+    data = ["//underactuated/models:double_pendulum.urdf"],
```

Comment:
I think this was the syntax that tricked me the other time I tried. I'll try again.